### PR TITLE
fixes #130 add tenderers field to contractingProcessSummary/tender

### LINKED ISSUE #130: Add a tenderers field to contractingProcessSummary/tender
CoST Ukraine's platform captures bidder information for the contracting processes associated with an infrastructure project.

Bidders can be added to the `project/parties` section, however there is no way of linking a bidder to a specific contracting process.

We should re-use the `tenderers` field from the OCDS `tender` object to link to the bidders from each contracting process.

### PR DESCRIPTION
**Pull request by pindec**
fixes #130

### CODE REVIEW

**Comment by duncandewhurst** [comment]
@pindec is this ready for review?

**Comment by pindec** [comment]
@duncandewhurst yes, sorry, didn't request one from you!

File: schema/project-level/project-schema.json [hunk 1]
```diff
@@ -306,6 +306,15 @@
               "description": "The number of parties who placed a bid during this contracting process.",
               "type": "number"
             },
+            "tenderers": {
+              "title": "Tenderers",
+              "description": "All parties who submit a bid on a tender. More detailed information on bids and the bidding organization can be provided using the bid extension.",
```
Comment (duncandewhurst) [inline schema/project-level/project-schema.json]:
```suggestion
              "description": "All parties who submit a bid on a tender. More detailed information on bids and the bidding organization can be provided using the bid extension in a linked OCDS release.",
```

**Review by duncandewhurst — CHANGES_REQUESTED** [review]
Thanks Charlie! Please add to the worked example JSON file and guidance page.

**Comment by pindec** [comment]
Will also need to update blank json file.

**Comment by pindec** [comment]
@duncandewhurst  I've updated the example file. Let me know if I also need to add a separate section in the example guidance page.

File: docs/examples/example.json [hunk 2]
```diff
@@ -191,7 +191,29 @@
             "url": "https://www.example.com"
           },
           "roles": [
-            "supplier"
+            "supplier",
```
Comment (duncandewhurst) [inline docs/examples/example.json:205]: Remove the 'supplier' role as this party is not listed in `contractingProcesses.summary.suppliers`

File: docs/examples/example.json [hunk 3]
```diff
@@ -191,7 +191,29 @@
...
+            "tenderer"
+          ]
+        },
+        {
+          "name": "Motorway Design Services PLC",
+          "id": "GB-COH-12345678",
+          "identifier": {
+            "scheme": "GB-COH",
+            "id": "22222222",
```
Comment (duncandewhurst) [inline docs/examples/example.json]: Update to match the identifier used in `party.id`

**Review by duncandewhurst — CHANGES_REQUESTED** [review]

**Comment by duncandewhurst** [comment]
> @duncandewhurst I've updated the example file. Let me know if I also need to add a separate section in the example guidance page.

Thanks @pindec! No need to add an extra section to the guidance page.

File: docs/examples/example.json (hunk 2 above)
Comment (pindec) [reply docs/examples/example.json:205]: The party with id `GB-COH-11111111` is listed in `contractingProcess.summary.suppliers` for ocid = ocds-a1b1c1-a410a80d-adc8-11e6-9901-0019b9f3037b so I think this is OK as-is?

Re-using the same URLs for each party doesn't help!

Comment (duncandewhurst) [reply docs/examples/example.json:205]: Sorry, I commented this on the wrong row. I'll add the comment in the correct place

File: docs/examples/example.json [hunk 4]
```diff
@@ -191,7 +191,29 @@
...
+          },
+          "address": {
+            "streetAddress": "567 High Street",
+            "locality": "Mareham",
+            "region": "Eastshire",
+            "postalCode": "ES22 3TW",
+            "countryName": "UK"
+          },
+          "roles": [
+            "supplier",
```
Comment (duncandewhurst) [inline docs/examples/example.json]: Remove 'supplier' role

**Review by duncandewhurst — CHANGES_REQUESTED** [review]

File: docs/examples/example.json (hunk 4 above)
Comment (pindec) [reply docs/examples/example.json]: Ah, thanks, now removed.

**Review by duncandewhurst — APPROVED** [review]

**Pull request merged**